src/main/java/com/renatusnetwork/momentum/commands/SquadCMD.java

```diff
@@ -0,0 +1,268 @@
+package com.renatusnetwork.momentum.commands;
+
+import com.renatusnetwork.momentum.Momentum;
+import com.renatusnetwork.momentum.data.levels.Level;
+import com.renatusnetwork.momentum.data.squads.Squad;
+import com.renatusnetwork.momentum.data.squads.SquadManager;
+import com.renatusnetwork.momentum.data.stats.PlayerStats;
+import com.renatusnetwork.momentum.utils.Utils;
+import net.md_5.bungee.api.chat.ClickEvent;
+import net.md_5.bungee.api.chat.HoverEvent;
+import net.md_5.bungee.api.chat.TextComponent;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.ConsoleCommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.Arrays;
+
+public class SquadCMD implements CommandExecutor {
+	@Override
+	public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+		if (sender instanceof ConsoleCommandSender) {
+			sender.sendMessage("This command cannot be run from console!");
+			return true;
+		}
+
+		SquadManager squadManager = Momentum.getSquadManager();
+		PlayerStats player = Momentum.getStatsManager().get((Player) sender);
+		Squad squad = player.getSquad();
+
+		int n = args.length;
+		if (label.equalsIgnoreCase("sqc")) {
+			if (squad == null)
+				noSquad(sender);
+			else if (n == 0)
+				player.sendMessage(Utils.translate("&3You have toggled &9&lSquad Chat &b" + (squadManager.toggleSquadChat(player) ? "on" : "off")));
+			else {
+				String msg = String.format("&9SqC &3%s &b%s", player.getDisplayName(), String.join(" ", Arrays.copyOfRange(args, 0, n)));
```

Comment:
Should this maybe be "SC" instead of "SqC" so that it's same style as Clan Chat's CC but the S for squads?